DIPTest/DIPTest/Navigator/Navigator.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  Navigator.swift
+//  DIPTest
+//
+//  Created by Uladzislau Abramchuk on 12/6/20.
+//
+
+import UIKit
+
+protocol Presentable {
+    func toPresent() -> UIViewController?
```

Comment:
тут `func` не нужен, также optional тоже не нужен
```suggestion
    var viewController: UIViewController { get }
```